scout/server/blueprints/variant/templates/variant/acmg.html

```diff
@@ -103,10 +103,15 @@ <h4>Evidence of {{ category }}</h4>
       </div>
     <!-- classification preview in the footer-->
       <div class="mt-3 fixed-bottom bg-light border">
-        <div class="text-center">
+        <div class="row">
+          <div class="col-3" style="font-size:1.5em">
+            <a href="https://clinicalgenome.org/docs/modeling-the-acmg-amp-variant-classification-gudielines-as-a-bayesian-classification-framework/" data-bs-toggle="tooltip" title="Classification based on the Bayesian score model in Tavtigian et al." rel="noopener noreferrer" target="_blank"><span id="temperature_span" class="badge"></span></a>
```

Comment:
Ah so it wasn't me that couldn't undestand, this time! 🤣

Reply:
Ah now I see it! 🥳

Reply:
No, since we had multiple PRs affecting the same pages the merges got a bit out of hand after a while. From that perspective, it is easier to clump several issues into one PR, but it also makes it harder to test and review.

Reply:
I hope that was the only thing that got botched - do say if something more seems to be missing!